module/sidebar/components/menu-component.js

```diff
@@ -298,4 +308,144 @@ export default class MenuComponent extends Component {
+            },
+            {
+                name: "Grapple",
+                ac: 4,
+                action: "Standard",
+                class: "Status",
+                range: "Melee, 1 Target",
+                effect: "You and the target each make opposed Combat or Athletics Checks. If you win, you and the target each become Grapples, and you gain Dominance in the Grapple.<br>&nbsp;<br>&nbsp;Pokemon and Trainers that are Grappled<br>&nbsp;» Are Vulnerable.<br>&nbsp;»Cannot take Shift Actions, or any actions that would cause them to Shift.<br>&nbsp;»Gain a -6 penalty to Accuracy rolls if targeting any-one outside the Grapple.<br>&nbsp;»Additionally, Grapply has other effects on whether the target has or doesn't have dominance.<br>&nbsp;<br>&nbsp;If a target begins their turn as part of a Grapple but with no Dominance, they may choose to contest the Grapple as a Full Action. If they do, all participants make opposed Combat or Athletics Checks. Whoever wins then may choose to either continue the Grapple and gain Dominance, or to end the Grapple<br>&nbsp;<br>&nbsp;If a target has the Phasing or Teleport Capability, they may use those to escape from a Grapple on their turn with no check required.<br>&nbsp;<br>&nbsp;If a Target Begins their turn as part of a Grapple and has Dominance, they may take one of the following actions as a Full action.<br>&nbsp;»End the Grapple<br>&nbsp;»Secure:They gain a +3 Bonus to the next opposed check they make in the Grapple.<br>&nbsp;»Attack: They may automatically hit with an Unarmed Struggle attack.<br>&nbsp;»Move: They Shift, dragging the other person in the grapple with them. The user's Movement Capability is lowered by the other grappler's Weight Class"
+            }
+
+        );
+
+        return maneuvers.map(m => m);   
+    }
+
+    async _sendManeuverToChat(maneuverName) {
+        let maneuver;
+        this._prepareManeuvers(this.state.actor).forEach(m => {
+            if (m.name == maneuverName) maneuver = m;
+        })
+
+        const messageData = mergeObject({
+            title: maneuver.name,
+            user: game.user._id,
+            action: maneuver.action,
+            trigger: maneuver.trigger,
+            ac: maneuver.ac,
+            class: maneuver.class,
+            range: maneuver.range,
+            effect: maneuver.effect,
+            sound: CONFIG.sounds.dice,
+            templateType: 'maneuver',
+        })
```

Comment:
You're not merging anything? Remove the mergeObject
```suggestion
        const messageData = {
            title: maneuver.name,
            user: game.user._id,
            action: maneuver.action,
            trigger: maneuver.trigger,
            ac: maneuver.ac,
            class: maneuver.class,
            range: maneuver.range,
            effect: maneuver.effect,
            sound: CONFIG.sounds.dice,
            templateType: 'maneuver',
        }
```